domains/parth.json

```diff
@@ -0,0 +1,17 @@
+{
+  "repo": "https://github.com/forrealdeveloper/Portfolio",
+
+  "owner": {
+    "username": "forrealdeveloper",
+    "email": "parthsali04@gmail.com"
+  },
+
+  "target": {
+    "record-type": {
+       "name": "TXT", 
```

Comment:
The TXT record should be placed in the actual record-type, so that should be replaced with the record you are requesting.

Reply:
I have made the suggested changes to the JSON file as per your feedback. Could you please review it again and let me know if any further modifications are needed? Thank you for your time and guidance!